examples/crm/src/companies/CompanyCard.tsx

```diff
@@ -59,9 +59,12 @@ const StyledMuiLink = styled(MuiLink)(({ theme }) => ({
     },
 }));
 
-export const CompanyCard = ({ record }: { record: Company }) => {
+export const CompanyCard = (props: { record: Company }) => {
     const [elevation, setElevation] = useState(1);
     const createPath = useCreatePath();
+    const record = useRecordContext<Company>(props);
```

Comment:
I think we should you should put a RecordContextProvider in `GridList` if we really want to use the record context. Seems overkill to me as we manually pass the record